gitlab/_version.py

```diff
@@ -3,4 +3,4 @@
 __email__ = "gauvainpocentek@gmail.com"
 __license__ = "LGPL3"
 __title__ = "python-gitlab"
-__version__ = "5.6.0"
+__version__ = "5.7.1"
```

Comment:
Semantic release will automatically update the version number. So should not modify this file.

Reply:
Sorry, I made a change because Python 3 detection requires a new version number. I'll either remove this change before the official merge or rebase to your latest version and then use your version number.